### PR TITLE
Debug log webhook

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This adds a nil-pointer check (that should only ever be triggered if someone manually sends crap to the webhook) and some debug logging so we can hopefully figure out what's causing the current issues.

I have not tested the log line. :crossed_fingers: 

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: config/kubermatic/templates/master-controller-manager-dep.yaml [hunk 1]
```diff
@@ -29,6 +29,7 @@ spec:
         - -internal-address=0.0.0.0:8085
         - -v=2
         - -logtostderr
+        - -log-debug=true
```
Comment (alvaroaleman) [inline config/kubermatic/templates/master-controller-manager-dep.yaml]: Are you sure this should be the default? This doesn't only affect the one log you added

Comment (xrstf) [reply config/kubermatic/templates/master-controller-manager-dep.yaml]: My thought was to let the debug log in place and disable the debug-logging later, but we can also do it on info level and change it to debug later.

Comment (alvaroaleman) [reply config/kubermatic/templates/master-controller-manager-dep.yaml]: I would more suggest to make it configurable in the chart, default to info and configure it to debug for us

**Pull request merged**